examples/binaries/calc-hash/over-blocks/src/code.rs

```diff
@@ -0,0 +1,71 @@
+use crate::Method;
+use gstd::{exec, msg, ActorId, BTreeMap, Decode, Encode, MessageId, Vec};
+use shared::{Package, PackageId};
+
+mod state {
+    use gstd::{ActorId, BTreeMap, Vec};
+    use shared::{Package, PackageId};
+
+    pub static mut THRESHOLD: Option<u64> = None;
+    pub static mut REGISTRY: BTreeMap<PackageId, Package> = BTreeMap::new();
+}
+
+#[no_mangle]
+unsafe extern "C" fn init() {
+    state::THRESHOLD = Some(msg::load().expect("Invalid threshold."));
+}
+
+#[gstd::async_main]
+async fn main() {
+    let method = msg::load::<Method>().expect("Invalid contract method.");
+
+    unsafe {
+        let threshold = state::THRESHOLD.expect("Threshold has not been set.");
```

Comment:
(Minor comment as best practice) I'd place this to the start of main just to clarify that if we run into some critical issue and don't have threshold set, we won't waste gas for decoding payload